File: IssueTracker.Data/Comment.cs
using System;
using System.Collections.Generic;

namespace IssueTracker.Data
{
    public class Comment
    {
        public long Id { get; set; }

        public long IssueId { get; set; }
        public long PersonId { get; set; }

        public string Message { get; set; }

        public bool Deleted { get; set; }

        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }

        public List<CommentChange> Changes { get; set; }
    }
}

File: issuetracker/app/assets/javascripts/models/comment.js
App.Comment = DS.Model.extend({
  issue: DS.belongsTo('App.Issue'),
  person: DS.belongsTo('App.User'),
  
  message: DS.attr('string'),
  deleted: DS.attr('boolean'),
  
  createdAt: DS.attr('date'),
  updatedAt: DS.attr('date'),
  
  changes: DS.hasMany('App.CommentChange')
});

App.RESTAdapter.map('App.Comment', {
  changes: { embedded: 'load' }
});